src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,53 @@
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import classNames from 'classnames';
+
+interface Props {
+  person: Person;
+}
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const { slug } = useParams();
+  const { name, sex, born, died, fatherName, motherName, mother, father } =
+    person;
+  const female = sex === 'f';
+  const notParents = '-';
```

Comment:
Create constants for magic values outside the component and name them in `UPPER_CASE`